app/js/module/dom.js

```diff
@@ -89,5 +89,15 @@ asciidoctor.browser.dom = ((document) => {
     return div.innerHTML;
   };
 
+  /**
+   * Set the appropriate viewport <meta> tag
+   */
+  module.setViewport = () => {
+    const viewportMeta = document.createElement('meta');
+    viewportMeta.name = "viewport";
+    viewportMeta.content = "width=device-width, initial-scale=1.0";
+    module.appendOnce(document.head, viewportMeta);
```

Comment:
`appendOnce` is using the element's id but the `meta` element does not have an `id` attribute so it won't work.
Since the `setViewport` function should be called only once (see next comment), you can do:

```js
document.head.appendChild(viewportMeta);
```

Reply:
I agree. I thought there might be some subtle functionality of `Element.id` that I wasn't aware of since all other calls to `Dom.appendOnce` follow don't appear to have explicit `id`s.

Reply:
> since all other calls to Dom.appendOnce follow don't appear to have explicit ids.

OH! Are you sure about that ? I didn't find any reference without an explicit id ?

Reply:
Ah, I didn't read the source properly. I see the id now! Silly me.